### ISSUE TITLE
Bootstrap - Site set to login both ways but shows Email for login

### ISSUE THREAD

**Issue by Data66PF** [moxi9/phpfox#778]
On the login, we show text of Email (above the input box) in bootstrap theme but the site is set for login both via email or username. This is confusing as it should say Email or Username

![loginemail](https://cloud.githubusercontent.com/assets/6908282/11588952/7a134428-9a52-11e5-84aa-6d9efd7e7d17.png)

V4.1.1 build 3

**Comment by Optiroot** [comment]
Temporary fix if someone uses Email and Usernames way, in pure CSS:

```
div.guest_login.header-login-form div.form-group:nth-child(1) label::after {
    content: " or username";
}
```
The high specification is to avoid issues with classes present inside the website!
Obviously this is an hack, but could be useful is someone needs it!

**Comment by Data66PF** [comment]
Thanks for the contribution @Optiroot :)

**Comment by robert9592** [comment]
Fixed in v4.1.2
